.github/workflows/push.yml

```diff
@@ -0,0 +1,41 @@
+on: push
+name: Build and Publish
+jobs:
+  dockerLint:
+    name: Docker Lint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
```

Comment:
```suggestion
```

This line is not needed and it's breaking the workflow.

Reply:
And btw, thanks for PR you cleared a lot of concepts for me :)

Reply:
Hmm, do you have an example of it? The workflow failed in my fork due to the branch not being master, but not on the checkout step.

I will try running the same workflow but on my fork's master and check that it passes

Reply:
Tried it on my fork's master, it seems to be working okay. Only failure is on Docker Login, which makes sense since I don't have the username/password.